capstone/src/main/webapp/youtubeScript.js

```diff
@@ -314,7 +314,7 @@ function perspectiveToxicityScale(attributeAverages) {
   const knoopScore = averageToxicityScore * 7000;
```

Comment:
Can this youtubeScript.js file be removed (probably just with `git rm`) now that the file have been renamed to youtube**Type**script (and `youtubeTypescript.js` is added to .gitignore)

Reply:
Yes, I've deleted it in "delete youtube script"